package.json

```
@@ -19,9 +19,9 @@
 		"eslint-plugin-vue": "9.23.0",
 		"prettier": "3.1.0"
 	},
-	"packageManager": "pnpm@8.15.4",
+	"packageManager": "pnpm@9.0.4",
```

The world of software is quick-moving. There's v9.0.5 already.


```suggestion
	"packageManager": "pnpm@9.0.5",
```